tests/e2e/lib.sh

```diff
@@ -630,18 +630,18 @@ wait_for_api() {
         API_PORT=443
     fi
     API_ENDPOINT="${API_HOSTNAME}:${API_PORT}"
-    METADATA_URL="https://${API_ENDPOINT}/v1/metadata"
-    info "METADATA_URL is set to ${METADATA_URL}"
+    PING_URL="https://${API_ENDPOINT}/v1/ping"
+    info "PING_URL is set to ${PING_URL}"
 
     set +e
     NUM_SUCCESSES_IN_A_ROW=0
     SUCCESSES_NEEDED_IN_A_ROW=3
     # shellcheck disable=SC2034
     for i in $(seq 1 60); do
-        metadata="$(curl -sk --connect-timeout 5 --max-time 10 "${METADATA_URL}")"
-        metadata_exitstatus="$?"
-        status="$(echo "$metadata" | jq '.licenseStatus' -r)"
-        if [[ "$metadata_exitstatus" -eq "0" && "$status" != "RESTARTING" ]]; then
+        pong="$(curl -sk --connect-timeout 5 --max-time 10 "${PING_URL}")"
```

Comment:
Is this a ping-pong joke or a typo. I'm ok either way.

Reply:
Only half-joke: https://github.com/stackrox/stackrox/blob/b85156909d09b5a2dc0aab07f4001b1eabb08821/proto/api/v1/ping_service.proto#L11-L17